src/components/dashboard/dashboard.service.ts

```diff
@@ -1255,6 +1275,22 @@ export class DashboardService implements OnDestroy {
             dimensions.height = this.getRowHeight();
         }
 
+        const minWidth = this.getColumnWidth() * widget.minColSpan + 1;
+        const widthSmallerThanAllowed = dimensions.width < minWidth;
```

Comment:
Just a suggestion, its often good practice to prefix boolean variables with `is` or `has` to indicate the type, e.g. `hasInvalidWidth` or `isBelowMinWidth` etc..